apcd-cms/src/apps/utils/apcd_database.py

```diff
@@ -660,6 +660,7 @@ def create_other_exception(form, sub_data):
     cur = None
     conn = None
     values = ()
+    values = ()
```

Comment:
Possible duplicate line

Reply:
Ah yes I think that was a hold over from a merge. Thank you!